src/components/customerPortal/PortalInvoicesList.tsx

```diff
@@ -76,11 +76,12 @@ gql`
 const mapStatusConfig = ({
   paymentStatus,
   paymentOverdue,
-  paymentDisputeLosable,
+  paymentDisputeLostAt,
 }: {
   paymentStatus: InvoicePaymentStatusTypeEnum
   paymentOverdue: boolean
-  paymentDisputeLosable: boolean
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  paymentDisputeLostAt: any | null
```

Comment:
Isn't it string | null?

Reply:
Oh yeah the type from the generated is any 😢

Reply:
I guess typing as string works!